clang/test/CheckedCRewriter/arrbothmulti2.c

```diff
@@ -1,10 +1,10 @@
-// RUN: cconv-standalone -base-dir=%S -alltypes -output-postfix=checkedALL2 %S/arrbothmulti1.c %s
-// RUN: cconv-standalone -base-dir=%S -output-postfix=checkedNOALL2 %S/arrbothmulti1.c %s
-//RUN: %clang -c %S/arrbothmulti1.checkedNOALL2.c %S/arrbothmulti2.checkedNOALL2.c
-//RUN: FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" --input-file %S/arrbothmulti2.checkedNOALL2.c %s
-//RUN: FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" --input-file %S/arrbothmulti2.checkedALL2.c %s
-//RUN: rm %S/arrbothmulti1.checkedALL2.c %S/arrbothmulti2.checkedALL2.c
-//RUN: rm %S/arrbothmulti1.checkedNOALL2.c %S/arrbothmulti2.checkedNOALL2.c
+// RUN: cconv-standalone -base-dir=%S -addcr -alltypes -output-postfix=checkedALL2 %S/arrbothmulti1.c %s
+// RUN: cconv-standalone -base-dir=%S -addcr -output-postfix=checkedNOALL2 %S/arrbothmulti1.c %s
+// RUN: %clang -c %S/arrbothmulti1.checkedNOALL2.c %S/arrbothmulti2.checkedNOALL2.c
```

Comment:
I guess I never looked carefully at the multi-file ones before. It seems like the first two lines above are running 3C on `...multi1.c` but then the third line is running `clang` on `...multi1.c` -- I guess you are assuming that that file is around because you converted first, with an earlier test?

Reply:
The syntax for llvm-lit isn't the clearest. The first line says: 
```
// RUN: cconv-standalone -base-dir=%S -addcr -alltypes -output-postfix=checkedALL2 %S/arrbothmulti1.c %s
```
The `%s` here is lit slang for "the current file", so the command above actually runs the conversion tool on _both_ arrbothmulti1.c and arrbothmulti2.c (arrbothmulti2 is represented by the %s).
The same thing is happening with the call to clang - I'm actually compiling both the files together - your reasoning about the availability of the files is spot on - I rely on the creation of the converted files in line 2 in order to compile it in line 3.